### ISSUE TITLE
dashboard content type listing portal

### ISSUE THREAD

**Issue by yogeshVU** [webgme/webgme-taxonomy#101]
A dashboard home page which serves a list of content types. 
Each content type would then redirect to the content type specific dashboard

**Comment by romancow** [comment]
@yogeshVU  @brollb

So I have an initial version of this in the linked branch. It's pretty basic. Thought you guys might want to take a look and give feedback.

I toyed with several approaches, and after some trial and error, this seemed to be the quickest solution. I tried creating a separate svelte bundle in the existing "Search" router, both being built by the same rollup config file, but ran into routing issues. Basically, I needed a json route to return the content types, but any path following the branch/tag/commit part of the url was being interpreted as a content-type. I tried some workarounds, but ultimately decided to just split it to a separate router.

Ultimately I think both this "dashboard" and the "Search" dashboard should be a single app, but it's gonna take some more work the integrate the two.

Also note currently you will have to build the svelte app (`npm run build`).
